### PR TITLE
specify all command args and use all games in game count

### PR DESCRIPTION
**Pull request by Sheikah45**

### CODE REVIEW

File: src/main/java/com/faforever/client/fa/ForgedAllianceService.java [hunk 1]
```diff
@@ -30,40 +31,46 @@
...
+    PlayerBean currentPlayer = playerService.getCurrentPlayer();
 
     Optional<LeaderboardRatingBean> leaderboardRating = Optional.of(currentPlayer.getLeaderboardRatings())
-        .map(rating -> rating.get(ratingType));
+        .map(rating -> rating.get(gameLaunchMessage.getLeaderboard()));
 
     float mean = leaderboardRating.map(LeaderboardRatingBean::getMean).orElse(0f);
     float deviation = leaderboardRating.map(LeaderboardRatingBean::getDeviation).orElse(0f);
 
+    int uid = gameLaunchMessage.getUid();
```
Comment (Askaholic) [inline src/main/java/com/faforever/client/fa/ForgedAllianceService.java:59]: Is there a reason this is extracted to a variable? I don't see it used except in the builder.

File: src/main/java/com/faforever/client/fa/ForgedAllianceService.java [hunk 2]
```diff
@@ -30,40 +31,46 @@
...
-        .faction(faction)
+        .faction(gameLaunchMessage.getFaction())
+        .mapPosition(gameLaunchMessage.getMapPosition())
+        .expectedPlayers(gameLaunchMessage.getExpectedPlayers())
+        .team(gameLaunchMessage.getTeam())
+        .gameOptions(gameLaunchMessage.getGameOptions())
         .clan(currentPlayer.getClan())
         .country(currentPlayer.getCountry())
+        .username(currentPlayer.getUsername())
+        .numberOfGames(currentPlayer.getNumberOfGames())
```
Comment (Askaholic) [inline src/main/java/com/faforever/client/fa/ForgedAllianceService.java:71]: So this is total across all games?

**Review by Askaholic — DISMISSED** [review]

File: src/main/java/com/faforever/client/fa/ForgedAllianceService.java (hunk 2 above)
Comment (Sheikah45) [reply src/main/java/com/faforever/client/fa/ForgedAllianceService.java:71]: Yes at least all ranked games

File: src/main/java/com/faforever/client/fa/ForgedAllianceService.java (hunk 1 above)
Comment (Sheikah45) [reply src/main/java/com/faforever/client/fa/ForgedAllianceService.java:59]: It is used to define the log file as well

**Pull request merged**